Testscripts/Linux/DetectLinuxDistro.sh

```diff
@@ -72,6 +72,9 @@ DetectDistro()
                 elif [[ "$tmp" =~ "SUSE Linux Enterprise Server 15" ]]; then
                     echo "SLES 15"
                     exitVal=0
+                elif [[ "$tmp" =~ "SUSE Linux Enterprise High Performance Computing 15 SP1" ]]; then
```

Comment:
can you make this more generic, like "SUSE Linux Enterprise High Performance Computing"?
That way we might catch any HPC images in the future, and not be hardcoded to v15 SP1.

Reply:
same edit here - https://github.com/LIS/LISAv2/pull/319/files#diff-2411aaddd5536a1d954db78a819c3f6fR1848
don't know why it won't let me comment on that line